### PR TITLE
bots: Move container tests to Python 3

### PR DESCRIPTION
**Pull request by martinpitt**

### CODE REVIEW

File: test/verify/kubelib.py [hunk 1]
```diff
@@ -23,7 +23,12 @@
 import sys
 import time
 
-import parent
+try:
+    import parent
+except ImportError:
+    # fallback for test/containers, where import is relative to test/verify
+    from . import parent
```
Comment (mvollmer) [inline test/verify/kubelib.py]: Can we use this form everywhere?

Comment (mvollmer) [reply test/verify/kubelib.py]: Actually, is kubelib.py used from somewhere outside of the Cockpit repo?  If not, I think we can simplify its own imports to just `import testlib`, and add `import parent` to `verify/check-openshift`.

The current pattern is that each test script starts of with `import parent` and the various modules such as `storagelib.py` can assume that the path has been set up correctly.

Comment (mvollmer) [reply test/verify/kubelib.py]:
```diff
diff --git a/test/verify/check-openshift b/test/verify/check-openshift
index 6a049b32b..109d0b4b2 100755
--- a/test/verify/check-openshift
+++ b/test/verify/check-openshift
@@ -21,6 +21,8 @@
 import os
 import unittest
 
+import parent
+
 from kubelib import *
 from testlib import *
 
diff --git a/test/verify/kubelib.py b/test/verify/kubelib.py
index 7e30d8f1b..89bb76370 100644
--- a/test/verify/kubelib.py
+++ b/test/verify/kubelib.py
@@ -23,17 +23,7 @@ import re
 import sys
 import time
 
-try:
-    import parent
-except ImportError:
-    # fallback for test/containers, where import is relative to test/verify
-    from . import parent
-parent  # pyflakes
-
-try:
-    import testlib
-except ImportError:
-    from common import testlib
+import testlib
 
 base_dir = os.path.dirname(os.path.realpath(__file__))
 
@@ -1666,5 +1656,3 @@ class RegistryTests(object):
         # also check with CLI
         testlib.wait(lambda: '2.11' in self.openshift.execute('oc get imagestream --namespace=marmalade sometags'))
         testlib.wait(lambda: 'latest' not in self.openshift.execute('oc get imagestream --namespace=marmalade sometags'))
-
-
```

Comment (martinpitt) [reply test/verify/kubelib.py]: Indeed, this makes much more sense. This  was probably moved erroneously when splitting out kubelib.py. Thanks! I ran a few tests locally, pushed.

**Pull request merged**